src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/services/UserService.kt

```diff
@@ -32,7 +34,8 @@ class UserService(
     val user = User(
       displayName = userRequest.displayName,
       password = userRequest.password,
-      email = userRequest.email
+      email = userRequest.email,
+      libraries = ArrayList() // TODO: decide if we want to have a few generic libraries created upon account creation
```

Comment:
I think we agreed on a default one of just all the games you add? I think it makes sense to just initialize a default library for everyone, avoids some null pointers

Reply:
right! will fix that. 

We may also want to have set empty libraries like Backlog etc. since that's common but that can be discussed down the line